dev/io.openliberty.io.smallrye.reactive.messaging-provider4/src/io/openliberty/microprofile/reactive/messaging30/internal/OLReactiveMessaging30Extension.java

```diff
@@ -71,7 +72,7 @@ void beforeBeanDiscovery(@Observes BeforeBeanDiscovery discovery, BeanManager be
         //addAnnotatedType(ContextDecorator.class, discovery, beanManager);
 
         //io.smallrye.reactive.messaging.providers.metrics - These depend on Micrometer so we're excluding them
-//        addAnnotatedType(MetricDecorator.class, discovery, beanManager);
+        addAnnotatedType(MetricDecorator.class, discovery, beanManager);
```

Comment:
move the comment down

Reply:
Updated comments about the decorators for future reference